scripts/etc/init.d/rcS

```diff
@@ -85,6 +85,25 @@ sanitize_inputfile()
     fi
 }
 
+install_files()
+{
+  file_to_read=$1
+  sanitize_inputfile file_to_read
+  grep -v "^#\|^$" "${file_to_read}" | while read -r line; do
```

Comment:
good idea... you could also go a bit further and use this regex:
`^[[:space:]]*#\|^[[:space:]]*$`

this will allow whitespace (just whitespace) in front of the hash sign... and it'll match lines with just whitespace as well...